inst/pages/98_exercises.qmd

```diff
@@ -580,12 +629,94 @@ Useful functions: nrow, ncol, dim, summary, table, quantile, unique, scater::add
 
 ### Prevalent and core taxonomic features
 
-1. Estimate prevalence for your chosen feature (row, taxonomic group)  
-2. Identify all prevalent features and subset the data accordingly   
-3. Report the thresholds and the dimensionality of the data before and after subsetting  
-4. Visualize prevalence  
+1. Estimate prevalence for your chosen feature (specific row and taxonomic group)
+```{r}
+#| label: prevalence-ex1
+#| eval: True
+#| code-fold: true
+#| code-summary: "Show solution"
+
+# First, merge the features by rank
+altExp(tse,"Phylum") <- mergeFeaturesByRank(tse, "Phylum") 
+
+# Then get the prevalence for a specific sample 
+getPrevalence(altExp(tse,"Phylum"), detection = 1/100, 
+                   sort = TRUE, assay.type = "counts",
+                   as_relative = TRUE)["Phylum:Proteobacteria"]
+```   
+2. Identify all prevalent features and subset the data accordingly.
+
+```{r}
```

Comment:
im just thinking if this should be simplified by not using altExp?

Reply:
I updated the exercice to use tse instead of using altExp. Both work completely fine and this might indeed a little bit of variation to the exercise.